### PR TITLE
Adding Unit Tests

### PR DESCRIPTION
**Pull request by wcomartin**
@niemyjski or @m-e-conroy please take a look

As Discussed in PR https://github.com/m-e-conroy/angular-dialog-service/pull/141

I don't have 100% coverage, i added tests for controllers and the "dialogs" service and provider, need to add testing for translating

### CODE REVIEW

**Comment by niemyjski** [comment]
@m-e-conroy Looks good to me.

File: tests/dialogs-controllers.tests.js [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+      beforeEach(function(){
+        $ctrlr('waitDialogCtrl', {
+          $scope: $scope,
+          $modalInstance: $modalInstance,
+          $translate: $translate,
+          data: {}
+        });
+      });
+
+      // it('should close when receiving event (dialogs.wait.complete) ', function(){
```
Comment (dougmoscrop) [inline tests/dialogs-controllers.tests.js]: Do you want to make these as it.skip instead? Or remove them?

**Comment by dougmoscrop** [comment]
:+1: from me too -- maybe kill the blocks of commented code, then I can merge.

File: tests/dialogs-controllers.tests.js (hunk 1 above)
Comment (wcomartin) [inline tests/dialogs-controllers.tests.js]: I actually forgot i had commented this out, i was working on testing the broadcasts but i couldn't get them working, so i commented it out for now i'll just remove them.

**Pull request merged**